pkg/service/egresstracking/service.go

```diff
@@ -128,5 +144,149 @@ func (s *EgressTrackingService) egressTrack(ctx context.Context, batchCID cid.Ci
 		return fmt.Errorf("invocation failed: %s", x.Message)
 	}
 
+	// Extract the consolidate invocation from the receipt's effects
+	effects := rcpt.Fx()
+	if effects != nil {
+		fork := effects.Fork()
```

Comment:
What would it mean for `effects` to be `nil`? Indicative of a developer error in the egress tracker?